Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:jessie
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+RUN apt-get -qq update && apt-get -qqy install python-pip python-dev
+RUN pip install junebug
+RUN mkdir logs
+
+EXPOSE 8080
+ENTRYPOINT ["jb", "--config=/app/example-config.yaml"]
```

Comment:
I'm wondering if `example-config` is the right name? Perhaps we should have `example/config.yaml`and just have `example` mounted at `app`?